tests/Microsoft.DotNet.Framework.Docker.Tests/WcfImageTests.cs

```diff
@@ -8,41 +8,36 @@
 
 namespace Microsoft.DotNet.Framework.Docker.Tests
 {
-    [Trait("Category", "wcf")]
+    [Trait("Category", ImageTypes.Wcf)]
     public class WcfImageTests : ImageTests
     {
-        private static readonly ImageDescriptor[] s_imageData = new ImageDescriptor[]
-        {
-            new ImageDescriptor { Version = "4.6.2", OsVariant = OsVersion.WSC_LTSC2016 },
-            new ImageDescriptor { Version = "4.7", OsVariant = OsVersion.WSC_LTSC2016 },
-            new ImageDescriptor { Version = "4.7.1", OsVariant = OsVersion.WSC_LTSC2016 },
-            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_LTSC2016 },
-            new ImageDescriptor { Version = "4.7.2", OsVariant = OsVersion.WSC_LTSC2019 },
-            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2016 },
-            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2019 },
-            new ImageDescriptor { Version = "4.8", OsVariant = OsVersion.WSC_LTSC2022 },
-            new ImageDescriptor { Version = "4.8.1", OsVariant = OsVersion.WSC_LTSC2022 },
-        };
-
         public WcfImageTests(ITestOutputHelper outputHelper)
             : base(outputHelper)
         {
         }
 
-        protected override string ImageType => "wcf";
+        protected override string ImageType => ImageTypes.Wcf;
+
+        public static IEnumerable<object[]> GetImageData() =>
+            ImageTestHelper.ApplyImageDataFilters(TestData.GetImageData(), ImageTypes.Wcf);
 
-        [SkippableTheory("3.5")]
+        private static bool IsSkippable(ImageDescriptor imageDescriptor) => imageDescriptor.Version == "3.5";
```

Comment:
Why not just apply this filter to the GetImageData and avoid all of the calls to `Skip.If(...)`

```
public static IEnumerable<object[]> GetImageData() =>
            ImageTestHelper.ApplyImageDataFilters(TestData.GetImageData(), ImageTypes.Wcf)
                .Where(descr => desc.Verion == "3.5");
```